README.md

```diff
@@ -102,8 +102,8 @@ if the object is defined by class definition.
 A route can have a list of actions in an HTTP path.
 ```python
 class ResourceRoute(Route):
```

Comment:
since 'Resource' was changed to 'Orders', shouldn't the class name be changed to something like OrdersRoute?

Reply:
Done. Also did it with API.